ur5_example.R

```diff
@@ -0,0 +1,13 @@
+#read csv data, add headers
+ur5=read.csv(file="ur5testresult_halfspeed_payload1.6lb_1 (1).csv",header=FALSE)
+ur5headers=scan("ur5testresult_header.csv", sep=',', what = "", quiet = TRUE)
+colnames(ur5)=ur5headers
+
+
+install.packages("scatterplot3d")
+library("scatterplot3d")

```

Comment:
No `library` in the middle of the script. In general, please look at the scripts' organization conventions:

From: https://google.github.io/styleguide/Rguide.xml
General Layout and Ordering
If everyone uses the same general ordering, we'll be able to read and understand each other's scripts faster and more easily.

Copyright statement comment
Author comment
File description comment, including purpose of program, inputs, and outputs
source() and library() statements
Function definitions
Executed statements, if applicable (e.g., print, plot)
Unit tests should go in a separate file named originalfilename_test.R.